cloudbuild/vendors_test.sh

```diff
@@ -31,31 +31,30 @@ popd
 
 # Deploy a cluster + topo
 pushd "$HOME"
-$cli deploy kne-internal/deploy/kne/kind-bridge.yaml
-$cli create kne/cloudbuild/presubmit/topology.textproto
+$cli deploy kne/deploy/kne/kind-bridge.yaml
```

Comment:
Should there be at least a 'set -x' here?

Reply:
there is on line 16